styles/normalize.css

```diff
@@ -0,0 +1,461 @@
+/*! normalize.css v5.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
its a different way of doing it that has trade offs. in general you want to avoid adding anything to your version control that you don't have to. If you can reference and external source you should. You could do this in many ways which all have tradeoffs. 

- You could link to a CDN with another `<link>` tag in the `<head>` of your HTML
- You could `npm i --save normalize.css` or `bower install normalize-css`
  - then either link to that locally installed file
  - or bundle that in with your code using something like webpack or browserify 

not saying you should do any of that for this goal but in general you should know your options